apps/core/migrations/0002_alter_dataset_status.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 4.1.3 on 2022-11-24 09:17
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("core", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="dataset",
+            name="status",
+            field=models.CharField(
+                choices=[
+                    ("pending", "Pending"),
+                    ("uploaded", "Uploaded"),
+                    ("extracted", "Extracted"),
+                    ("errored", "Errored"),
+                ],
+                default="pending",
+                max_length=50,
+            ),
+        ),
+    ]
```

Comment:
Let's create a single migration file here. Remove the old migrations and create a new one with these changes applied.

Reply:
done.